test/integration/smoke/test_list_ids_parameter.py

```diff
@@ -194,10 +194,7 @@ def setUpClass(cls):
         cls._cleanup = [
                         cls.disk_offering,
                         cls.account,
-                        cls.service_offering,
-                        cls.snapshot_1,
-                        cls.snapshot_2,
-                        cls.snapshot_3
+                        cls.service_offering
```

Comment:
I've tried to fix this, by reordering the items by ownership here: 
https://github.com/apache/cloudstack/pull/1502/commits/6135f6d98fa6b363b0468c17a8e713aeb5054437